backend/app/tests/services/test_ai_model_selection.py

```
@@ -0,0 +1,123 @@
+        
+        # 测试各种情况的优先级
+        assert select_model("summary.j2") == "or-deepseek-r1"
+        assert select_model("labels.j2") == "deepseek-v3-ensemble"
+        assert select_model("unknown.j2") == settings.DEFAULT_LLM_MODEL
+        assert select_model("summary.j2", "custom-model") == "custom-model"
+        assert select_model("unknown.j2", "custom-model") == "custom-model"
+
+    @pytest.mark.asyncio
+    async def test_different_templates_different_models(self, chat_service):
+        """测试不同模板确实使用不同模型"""
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)